docs/contributing.rst

```diff
@@ -2,22 +2,25 @@
 Contributing
 ############
 
-Help is always appreciated!
+Help is always appreciated and there are plenty of ways to contribute to Solidity.
 
-To get started, you can try :ref:`building-from-source` in order to familiarize
-yourself with the components of Solidity and the build process. Also, it may be
-useful to become well-versed at writing smart-contracts in Solidity.
+In particular, we appreciate support in the following areas:
```

Comment:
Word repetition: appreciate

Reply:
Changed to `help is always welcome`. Hope that works. : )